packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -50,8 +50,17 @@ export class AccountField extends PureComponent {
       showLabel: t('accountForm.password.show')
     }
     switch (type) {
+      case 'date':
+        return (
+          <Field
+            {...fieldProps}
+            placeholder={t(`fields.${name}.placeholder`, {
+              _: t(`default.dateFormat`).toLowerCase()
+            })}
+          />
+        )
       case 'dropdown':
-        return <Field {...sanitizeSelectProps(fieldProps)} />
```

Comment:
Is it related to this PR?

Reply:
It isn't and I should have see it. Thanks !